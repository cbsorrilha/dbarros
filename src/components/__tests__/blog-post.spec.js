import React from "react"
import renderer from "react-test-renderer"
import BlogPost from '../blog-post';


describe('Snapshot testing the BlogPost', () => {
    it("renders correctly", () => {
        const props = {
            slug: "slug",
            date: "date",
            description: "description",
            image: "image",
            title: "title"
        }
        const tree = renderer.create(<BlogPost {...props} />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
