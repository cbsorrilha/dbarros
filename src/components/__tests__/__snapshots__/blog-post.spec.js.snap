// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing the BlogPost renders correctly 1`] = `
<article
  className="sc-AykKD kRTVwG"
>
  <a
    href="slug"
    style={
      Object {
        "boxShadow": "none",
      }
    }
  >
    <div
      className="sc-AykKC dejEKn"
    >
      <img
        alt="description"
        src="image"
      />
    </div>
    <div
      className="sc-AykKE DxDpd"
    >
      <header>
        <h3>
          title
        </h3>
        <small>
          date
        </small>
      </header>
      <section>
        <p
          dangerouslySetInnerHTML={
            Object {
              "__html": "description",
            }
          }
        />
      </section>
    </div>
  </a>
</article>
`;
